snapd-glib/requests/snapd-request.c

```diff
@@ -61,7 +61,7 @@ void
 _snapd_request_set_source_object (SnapdRequest *request, GObject *object)
 {
     SnapdRequestPrivate *priv = snapd_request_get_instance_private (SNAPD_REQUEST (request));  
-    priv->source_object = g_object_ref (object);
+    priv->source_object = object ? g_object_ref (object) :  NULL;
```

Comment:
This doesn't seem useful - object will always be non-null in this case.

Reply:
Just wanted to be avoid problems that could arise in the future.